src/models/comment.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const Comment = sequelize.define('Comment', {
+    userId: DataTypes.INTEGER,
+    articleSlug: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      references: {
+        model: 'Articles',
+        key: 'slug'
+      },
+      onUpdate: 'CASCADE',
+      onDelete: 'CASCADE'
+    },
+    parentCommentId: {
+      type: DataTypes.INTEGER,
+      allowNull: true
+    },
+    body: DataTypes.STRING
+  }, {});
+  Comment.associate = function(models) {
+    // associations can be defined here
+    Comment.belongsTo(models.user,{
+      foreignKey:'id',as:'commentAuthor'
+    });
```

Comment:
@nshutijonathan I thought the Article model has a one to many relationship with the Comment model. What is your opinion about this?

Reply:
In my comment models, I have this  them Check very well references: {
        model: 'Articles',
        key: 'slug'
      },
      onUpdate: 'CASCADE',
      onDelete: 'CASCADE'
    },

Reply:
Cool. I understand now. Thank you for the clarification